### PR TITLE
Pricing page update

### PR DESCRIPTION
**Pull request by teallarson**
## Change description

Updates to our pricing page reflecting our new pricing structure.  Adds clarity for users RE: exact costs, what constitutes a "synced record".
Screen Shot 2021-11-30 at 8 17 06 PM

These are based on a combination of the designs that were created by Will and myself.  Some things that will change as the site-wide theme is updated in the coming days/weeks:
- Background will go from the current off-white to white
- Buttons become pill-shaped
- Primary color (button) changes from blue --> mint

This PR also changes the checkboxes on our comparison pages to the new purple color to coordinate with our new scheme and updates the `info` color in our color scheme to that which our designer gave us.

Description here

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/pricing.module.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.pricingList {
```
Comment (teallarson) [inline pages/pricing.module.scss:1]: @krishnaglick recommended we move towards using modular SCSS for items that are only used on a single page, so I've started us out.

File: pages/pricing.tsx [hunk 2]
```diff
@@ -46,100 +36,107 @@ export default function PricingPage() {
...
+                 shadow-lg h-100 text-start"
+                bg="info"
+                style={{ minHeight: "468px" }}
               >
                 <Card.Body>
-                  <Card.Title>Standard Cloud</Card.Title>
+                  <div className="h3">Standard Cloud</div>
                   <h5>Starting at $150/month</h5>
-                  For companies who want to sync up to 100k Records to their
+                  For companies wanting to sync up to 1MM Records to their
```
Comment (andyjih) [inline pages/pricing.tsx]:
```suggestion
                  For companies wanting to sync up to 1,000,000 Records to their
```

File: pages/pricing.tsx [hunk 3]
```diff
@@ -46,100 +36,107 @@ export default function PricingPage() {
...
                   destinations.
                   <br />
                   <br />
-                  <div style={boxTextStyle}>
+                  <div className={styles.pricingList}>
                     <b>Features include:</b>
-                    <li>Up to 100k Records synced/month</li>
-                    <li>Access to all Sources and Destinations</li>
+                    <li>Up to 1MM Records synced/month</li>
```
Comment (andyjih) [inline pages/pricing.tsx]:
```suggestion
                    <li>Up to 1,000,000 Records synced/month</li>
```

File: pages/pricing.tsx [hunk 4]
```diff
@@ -46,100 +36,107 @@ export default function PricingPage() {
...
                   For enterprises with larger data sets or unique use cases.
                   <br />
                   <br />
-                  <div style={boxTextStyle}>
+                  <div className={styles.pricingList}>
                     <b>Features include:</b>
-                    <li> &gt; 100k Records synced/month</li>
-                    <li>Business hours support</li>
-                    <li>Enhanced support response time SLAs</li>
+                    <li> &gt; 1MM synced Records/month</li>
```
Comment (andyjih) [inline pages/pricing.tsx]:
```suggestion
                    <li> &gt; 1,000,000 synced Records/month</li>
```

File: pages/pricing.tsx [hunk 5]
```diff
@@ -160,371 +157,379 @@ export default function PricingPage() {
...
+                          </td>
+                        </tr>
+                        <tr className="w-100 mx-auto">
+                          <td className="border-bottom-1 border-bottom-light">
+                            300K synced Records: $300/mo
+                          </td>
+                        </tr>
+                        <tr className="w-100 mx-auto">
+                          <td className="border-bottom-0">
+                            1MM synced Records: $600/mo
```
Comment (andyjih) [inline pages/pricing.tsx]:
```suggestion
                            1,000,000 synced Records: $600/mo
```

File: pages/pricing.tsx [hunk 6]
```diff
@@ -160,371 +157,379 @@ export default function PricingPage() {
...
+                          </td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </td>
+                  <td>
+                    &gt; 1MM synced Records:
```
Comment (andyjih) [inline pages/pricing.tsx]:
```suggestion
                    &gt; 1,000,000 synced Records:
```

File: pages/pricing.tsx [hunk 7]
```diff
@@ -46,100 +36,107 @@ export default function PricingPage() {
...
-                bg="secondary"
-                text="white"
+                className="mx-auto p-3 shadow-lg h-100 col-12"
+                bg="white"
+                style={{ minHeight: "468px" }}
               >
-                <Card.Body>
-                  <Card.Title>Enterprise Cloud</Card.Title>
+                <Card.Body className="d-flex mx-auto align-items-start flex-column">
+                  <div className="h3">Enterprise Cloud</div>
```
Comment (andyjih) [inline pages/pricing.tsx:123]: This div's text is centered, but the other cards' divs are left-aligned. Should probably be consistent?

**Review by andyjih — CHANGES_REQUESTED** [review]
Overall looks good! Made some suggestions. 👍

File: pages/pricing.tsx (hunk 7 above)
Comment (teallarson) [reply pages/pricing.tsx:123]: Ah ha!  Yep.

**Review by andyjih — APPROVED** [review]

**Pull request merged**